build/windows/installer/scripts/ConfigParseErrorLogger.rb

```diff
@@ -0,0 +1,21 @@
+#!/usr/local/bin/ruby
```

Comment:
this file can be removed. This is the same as the one in the plugins. Should be copied from that folder

Reply:
> this file can be removed. This is the same as the one in the plugins. Should be copied from that folder. Or are u keeping this here because of Docker Context issue we encounterd?
> moved to common->installer->scripts all the common files.